apk-generator/v2/scripts/build_apk.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
-./gradlew assembleRelease --info
+./gradlew assemble$3 --info
 echo "signing"
-jarsigner -keystore ${KEYSTORE_PATH} -storepass ${KEYSTORE_PASSWORD} app/build/outputs/apk/app-googleplay-release-unsigned.apk ${KEY_ALIAS}
+jarsigner -keystore ${KEYSTORE_PATH} -storepass ${KEYSTORE_PASSWORD} app/build/outputs/apk/app-$2-release-unsigned.apk ${KEY_ALIAS}
```

Comment:
$2 is lowercase @niranjan94 as we get the buildType from the the js file file which as you can see has name "googleplay" and "fdroid"